tensorboard/examples/plugins/example_raw_scalars/tensorboard_plugin_example/static/index.js

```diff
@@ -0,0 +1,72 @@
+// Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ==============================================================================
+
+import * as Model from './model.js';
+
+/**
+ * The main entry point of any TensorBoard iframe plugin.
+ */
+export async function render() {
+  const runs = await Model.getRuns();
+  if (!runs) {
+    document.body.innerHTML = `<h1>No runs found.</h1>`;
+    return;
+  }
+
+  document.body.innerHTML = `
```

Comment:
prefer to use `appendChild` than innerHTML.

Reply:
+1 to @stephanwlee’s comment. Making tradeoffs for concision of docs is
reasonable, but keep in mind that this will encourage our users to make
those same tradeoffs, and many users may not realize the implications.
If the tradeoff is that the code is a bit less elegant or clean, that’s
probably fine; if the tradeoff is that our users’ applications now have
security vulnerabilities by default, that’s certainly not. (My $0.02.)
Thanks for changing.